src/nn/wasm/src/bind/src/binding.cpp

```diff
@@ -0,0 +1,342 @@
+#include <emscripten/bind.h>
+#include <emscripten/val.h>
+
+#include "external/tensorflow/tensorflow/lite/kernels/internal/types.h"
+#include "external/tensorflow/tensorflow/lite/kernels/internal/optimized/optimized_ops.h"
+#include "external/tensorflow/tensorflow/lite/kernels/internal/reference/reference_ops.h"
+#include "external/tensorflow/tensorflow/lite/kernels/internal/optimized/depthwiseconv_float.h"
+
+#include <vector>
+#include <cmath>
+#include <iostream>
+
+using namespace emscripten;
+using namespace tflite;
+
+namespace binding_utils {
+  struct ReshapeParams {
+    uint32_t size_count;
+  };
+
+  // Factors set and get function
+  val getRuntimeShapeDimensions(const RuntimeShape& shape) {
+    return emscripten::val(
+      emscripten::typed_memory_view(shape.DimensionsCount(),
+                                    shape.DimsData()));
+  }
+
+  void setRuntimeShapeDimensions(RuntimeShape& shape, val js_dims) {
```

Comment:
I guess we don't need this method as `getRuntimeShapeDimensions` returns the `Uint32Array` of the `DimsData`. Could you test whether you can assign value to the `Uint32Array` directly?

Reply:
Do we need resize the dims?